DisCatSharp/Net/Abstractions/Rest/RestGuildScheduledEventPayloads.cs

```diff
@@ -0,0 +1,154 @@
+        [JsonProperty("description", NullValueHandling = NullValueHandling.Ignore)]
+        public Optional<string> Description { get; set; }
+
+        /// <summary>
+        /// Gets or sets the privacy level of the scheduled event.
+        /// </summary>
+        [JsonProperty("privacy_level")]
+        public ScheduledEventPrivacyLevel PrivacyLevel { get; set; }
+
+        /// <summary>
+        /// Gets or sets the time to schedule the scheduled event.
+        /// </summary>
+        [JsonProperty("scheduled_start_time")]
+        public DateTimeOffset ScheduledStartTime { get; internal set; }
+
+        /// <summary>
+        /// Gets or sets the time when the scheduled event is scheduled to end.
+        /// </summary>
+        [JsonProperty("scheduled_end_time")]
+        public Optional<DateTimeOffset?> ScheduledEndTime { get; internal set; }
+
+        /// <summary>
+        /// Gets or sets the entity type of the scheduled event.
+        /// </summary>
+        [JsonProperty("entity_type")]
+        public ScheduledEventEntityType EntityType { get; set; }
+
+        /// <summary>
+        /// Gets or sets the image as base64.
+        /// </summary>
+        [JsonProperty("image", NullValueHandling = NullValueHandling.Ignore)]
+        public string ImageBase64 { get; set; }
+    }
+
+    /// <summary>
+    /// The rest guild sheduled event modify payload.
+    /// </summary>
+    internal class RestGuildSheduledEventModifyPayload
+    {
+        /// <summary>
+        /// Gets or sets the channel id.
+        /// </summary>
+        [JsonProperty("channel_id")]
+        public Optional<ulong?> ChannelId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the entity metadata.
+        /// </summary>
+        [JsonProperty("entity_metadata")]
+        public Optional<DiscordScheduledEventEntityMetadata> EntityMetadata { get; set; }
+
+        /// <summary>
+        /// Gets or sets the name.
+        /// </summary>
+        [JsonProperty("name")]
+        public Optional<string> Name { get; set; }
+
+        /// <summary>
+        /// Gets or sets the description.
+        /// </summary>
+        [JsonProperty("description")]
+        public Optional<string> Description { get; set; }
+
+        /// <summary>
+        /// Gets or sets the privacy level of the scheduled event.
+        /// </summary>
+        [JsonProperty("privacy_level")]
+        public Optional<ScheduledEventPrivacyLevel> PrivacyLevel { get; set; }
+
+        /// <summary>
+        /// Gets or sets the time to schedule the scheduled event.
+        /// </summary>
+        [JsonProperty("scheduled_start_time")]
+        public Optional<DateTimeOffset> ScheduledStartTime { get; internal set; }
+
+        /// <summary>
+        /// Gets or sets the time when the scheduled event is scheduled to end.
+        /// </summary>
+        [JsonProperty("scheduled_end_time")]
+        public Optional<DateTimeOffset?> ScheduledEndTime { get; internal set; }
```

Comment:
Consider just dropping Optional<T> here and using `NullValueHandling = NullValueHandling.Ignore` instead.